app/views/tools/show.html.erb

```diff
@@ -0,0 +1,151 @@
+<% content_for :header_title do %>
+  <h2><%= fetch_entry_id(@tool, 'tool') %></h2>
+  <p class="subtitle"><%= display_header_subtitle(@tool, 'tool') %></p>
+<% end %>
+
+<% if @user_permissions == 'none' %>
+  <div class="eui-banner--danger">
+    <p class="eui-banner__message">
+      <%= "You don't have the appropriate permissions to #{@record_action} this tool." %>
+    </p>
+  </div>
+  <div class="no-access">
+    <h3 class="access-title">Access Denied</h3>
+    <p class="access-message">
+      It appears you do not have access to <%= @record_action %> this content.<br />
+      If you feel you should have access, please check with your provider manager or ensure you are logged into the correct provider.
+    </p>
+  </div>
+<% elsif @user_permissions == 'wrong_provider' %>
+  <div class="eui-banner--warn">
+    <p class="eui-banner__message">
+      <%= link_to("You need to change your current provider to #{@record_action} this tool. Click here to change your provider.",
+        "#", id: "change-current-provider-banner-link",
+        data: { "provider": @provider_id, action_link: "change-provider-tool-#{@record_action}" }) %>
+    </p>
+    <%# TODO: this method does not exist yet. It should be created and used when %>
+    <%# additional actions are added to published Tool records %>
+    <%#= render_change_provider_tool_action_link(@record_action, @concept_id, @revision_id) %>
+  </div>
+<% end %>
+
+<% if @unsupported_version %>
+  <%# with MMT-1559 see if this can be refactored into a partial %>
```

Comment:
I'm pretty interested in partialing out as much of this as we can at some point.  These erbs are pretty redundant across V/S/T.